ignite/engine/events.py

```diff
@@ -270,6 +270,7 @@ def __init__(self, **kwargs):
         self.metrics = {}
         self.dataloader = None
         self.seed = None
+        self.times = {Events.EPOCH_COMPLETED.name: 0.0, Events.COMPLETED.name: 0.0}
```

Comment:
Maybe, we need to init values as `None`, such that if, for example, user request values of `Events.COMPLETED.name` it wont say that run took 0.0 seconds... Thoughts ?

Reply:
I agree. This makes the wiring more intentional, too.